strings/power_for_large_numbers_represented_as_strings.cpp

```diff
@@ -0,0 +1,45 @@
+/**
+ * \file
+ * \brief The gfg
+ *modulo-power-for-large-numbers-represented-as-strings
+ */
+#include <iostream> 
+using namespace std; 
+#define ll long long int 
+const ll MOD = 1e9 + 7; 
+// Returns modulo exponentiation for two numbers 
+// represented as long long int. It is used by 
+// powerStrings(). Its complexity is log(n) 
+ll powerLL(ll x, ll n) { 
+	ll result = 1; 
+	while (n)
+	 { 

```

Comment:
```suggestion
	while (n) {
```
This looks clean.

Reply:
I think this can be automatically handled through CI Workflow `code-formatter`.

Reply:
Yeah, but I think it would be a good practice if he/she could do this on their own.